src/views/v2/tickets.php

```diff
@@ -72,6 +72,7 @@ class="tribe-tickets"
 
 	</form>
 
-	<?php $this->template( 'v2/modal/target' ); ?>
+	// @todo Convert this into an action.
+	<?php // $this->template( 'v2/modal/target' ); ?>
```

Comment:
:warning: This comment is 67% valid code; is this commented out code?

<sub>hash: 2091623ddae919b909d34c7c132ca594</sub>